src/SFA.DAS.ApplyService.Data/ApplyRepository.cs

```diff
@@ -40,10 +40,10 @@ public async Task<Guid> StartApplication(Guid applicationId, ApplyData applyData
             using (var connection = new SqlConnection(_config.SqlConnectionString))
             {
                 return await connection.QuerySingleAsync<Guid>(
-                    @"INSERT INTO Apply (ApplicationId, OrganisationId, ApplicationStatus, ApplyData, AssessorReviewStatus, CreatedBy, CreatedAt)
+                    @"INSERT INTO Apply (ApplicationId, OrganisationId, ApplicationStatus, ApplyData, CreatedBy, CreatedAt)
```

Comment:
Initial assessor review status should be set here

Reply:
We'll need to do it with all other Status columns. I removed them as the database will put default ones in

Reply:
Just looked at this... it's using ApplicationStatus from the wrong Entity. I'm refusing to add the other statuses until this is properly sorted out